File: 2st/Yeonwoo/l로 만들기.py
def solution(myString):
    answer = ''
    ar = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k"]
    for n in myString:
        if n in ar:
            answer += "l"
        else:
            answer += n
    return answer

File: 3st/Juram/programmer/l로 만들기.java
class Solution {
    public String solution(String myString) {
        StringBuilder answer = new StringBuilder();
        for(int i=0; i<myString.length(); i++){
            char ch = myString.charAt(i);
            if(ch < 'l') answer.append('l');
            else answer.append(ch);
            
        }
        return answer.toString();
    }
}